main/src/ca/uwaterloo/flix/language/phase/Safety.scala

```diff
@@ -345,6 +345,9 @@ object Safety {
       case Expression.FixpointProject(_, exp, _, _, _, _) =>
         visit(exp)
 
+      case Expression.Instanceof(exp, _, _, _, _, _) =>
```

Comment:
We could check that you don't write impossible things like `123 instanceof String`, but maybe it is not worth it?